tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -512,3 +519,8 @@ Object.Base {
 	}
 
 }
+
+IncludeByKey.SSP0_MIXER_2LEVEL {
+	"0"	"platform/intel/nocodec-ssp0-1level.conf"
```

Comment:
@lyakh there's pending PR in alsa-utils that allows us to include short conf blocks without vreating a new conf file. https://github.com/alsa-project/alsa-utils/pull/187. It will be good to use that after it is merged